cabinets/cabinet.py

```diff
@@ -41,27 +56,36 @@ def delete(cls, path):
     @classmethod
     @abstractmethod
     def _read_content(cls, path) -> bytes:
-        pass
+        pass  # pragma: no cover
 
     @classmethod
     @abstractmethod
     def _create_content(cls, path, content):
-        pass
+        pass  # pragma: no cover
 
     @classmethod
     @abstractmethod
     def _delete_content(cls, path):
-        pass
+        pass  # pragma: no cover
+
+
+class InvalidURIError(Exception):
+    pass
```

Comment:
Do we want to organize this up with the other custom Exception? or even in a different module?